pages/common/funzip.md

```diff
@@ -0,0 +1,12 @@
+# funzip
+
+> Print the content of the first (non-directory) member in an archive without extraction.
+> More information: <https://manned.org/funzip>.
+
+- Print the content of the first member in a `.zip` archive:
+
+`funzip {{path/to/archive}}`
```

Comment:
```suggestion
`funzip {{path/to/archive.zip}}`
```
Since you are specifying `.zip` archive shouldn't this be `path/to/archive.zip`?